src/components/Details/TranscriptSearch/TranscriptSearch.tsx

```diff
@@ -12,6 +12,7 @@ import { strings } from "../../../assets/LocalizedStrings";
 import { fontSizes } from "../../../styles/fonts";
 import { screenWidths } from "../../../styles/mediaBreakpoints";
 import cleanText from "../../../utils/cleanText";
+import { varietyRoles } from "../../../stories/model-mocks/role";
```

Comment:
You'll want to clean that up, the linter says it's never used.

Reply:
Just fixed this! It was there before I added my code, not sure how it passed the lints previously